public/locales/en/email.json

```diff
@@ -20,7 +20,7 @@
     "dont-receive": "If you don't receive an email, you can:",
     "dont-receive-list": {
       "item-1": "review the information you entered and try again or",
-      "item-2": "try again tomorrow — we update our records daily"
+      "item-2": "try again tomorrow—we update our records daily"
```

Comment:
Is that wanted?

Reply:
Yes, client requested spaces removed. Just didn't call it out in the PR description.